trackers/node-tracker/api-extractor.json

```diff
@@ -330,14 +330,11 @@
       "default": {
         "logLevel": "warning"
         // "addToApiReportFile": false
-      }
+      },
 
-      // "ae-extra-release-tag": {
-      //   "logLevel": "warning",
-      //   "addToApiReportFile": true
-      // },
-      //
-      // . . .
+      "ae-missing-release-tag": {

```

Comment:
I added it to the browser-tracker because theres a couple of things I don't want to export on purpose to simplify the API slightly but those don't exist on the node-tracker so I thought I'd leave it out as we're more likely to catch errors if its not here. Happy for it to end up here in a future release if we feel we need it though.